lib/registries/addon/components/registries-metadata/component.ts

```diff
@@ -27,13 +27,15 @@ export default class RegistriesMetadata extends Component {
     extendedFields?: boolean;
 
     @tracked provider?: RegistrationProviderModel;
+    @tracked hideCategories = true;
     // Private properties
     expandCitations = false;
 
     @task({ on: 'didReceiveAttrs'})
     @waitFor
     async fetchProvider() {
         this.provider = await this.registration?.provider;
+        this.hideCategories = true;
```

Comment:
Do you need hideCategories to be tracked and set it in fetchProvider, or could you leave it untracked and only set it at declaration?

Reply:
Yeah, let's keep it simple for now. I have a feeling it's more likely that we'll get rid of categories entirely rather than make it dependent on something else.